src/Place/Pages/PPA002Place/PlaceController.ts

```diff
@@ -0,0 +1,22 @@
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack/lib/typescript/src/types';
+
+import { PlaceControllerInterface, PlacePropsInterface } from 'spelieve-common/lib/Interfaces';
+
+import { BottomTabParamList } from '@/App';
+
+export const PPA002PlaceController = ({ place_id, language }: PlacePropsInterface): PlaceControllerInterface => {
```

Comment:
PlacePropsInterface から削除したから修正お願い！
```suggestion
export const PPA002PlaceController = ({ place_id }: PlacePropsInterface): PlaceControllerInterface => {
```

Reply:
[3dcf661](https://github.com/Ayato-kosaka/spelieve/pull/326/commits/3dcf6619e1529e2f42f42e2b6b7402c72fa12097)